INSTALL.md

```diff
@@ -26,33 +25,41 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are four scripts in the the `bin/` directory that are use to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
+3. `accumulo-single`- Manages Accumulo on a single node
+4. `accumulo-cluster` - Manages Accumulo on a cluster
```

Comment:
Is running a single-node not just a cluster of one node? Do we want to make that distinction?